values.yaml

```diff
@@ -67,4 +53,1543 @@ affinity: {}
 storage:
   storageClass: ""
   size: "1Gi"
-  accessModes: [ReadWriteOnce]
\ No newline at end of file
+  accessModes: [ReadWriteOnce]
+
+copyparty:
+  livenessProbe:
+    httpGet:
+      path: /
+      port: http
+  readinessProbe:
+    httpGet:
+      path: /
+      port: http
+  autoscaling:
+    enabled: false
+    minReplicas: 1
+    # @TODO: Clarify if Copyparty works properly in a ReadWriteMany Scenario
+    maxReplicas: 1
+    targetCPUUtilizationPercentage: 80
+    # targetMemoryUtilizationPercentage: 80
+  users:
+    user1: password
+    user2: correcthorsebatterystaple
+  groups:
+    # @TODO: Does user existence need to be verified?
+    group1:
+      - user1
+      - user2
+      - user3
+
+  
+  volumes:
+    - volumeName1:
+      httpURL: /the/url/to/share/this/volume/on/
+      mountPath: /the/actual/filesystem/path/
+      existingClaim: ""
+      storageClass: "longhorn-nvme"
+      resources:
+        requests:
+          storage: 2Gi
+        limits:
+          # @TODO: Sync with vmaxb
+          storage: 3Gi
+      # @TODO: Move to config since copyparty can't do RWX anyway (or should not at least)
+      accessModes:
+        - ReadWriteOnce
+      permissions:
+        users:
+          # @TODO: Clarify whether options can only be globbed for groups, or also users, and whether users and groups can be mixed
+          read:
+            - user1
+            - user2
+          write: 
+            - user1
+            - user2
+          move:
+            - user1
+            - user2
+          delete:
+            - user1
+            - user2
+          dotfilesVisible:
+            - user1
+            - user2
+          # @TODO: Explain what exactly the next two do!
```

Comment:
It's explained here https://github.com/9001/copyparty?tab=readme-ov-file#accounts-and-volumes under permissions: 
g (get): only download files, cannot see folder contents or zip/tar
G (upget): same as g except uploaders get to see their own [filekeys](https://github.com/9001/copyparty?tab=readme-ov-file#filekeys) (see fk in examples below)